pwndbg/wrappers/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.which
+
+
+class OnlyWithCommand(object):
+    def __init__(self, func):
+        self.func = func
```

Comment:
Another thing is all of other `Only***` are actually functions - it might be better to keep it consistent.